docs/_includes/date-picker.html

```diff
@@ -0,0 +1,282 @@
+<div id="date-picker">
+    {% include section-header.html name="date-picker" version=page.versions.date-picker %}
+
+    <p>The date picker component is used to display a calendar from which dates can be selected, either as single items or as a range.</p>
+
+    <h3>Single Pane</h3>
+
+    <button class="btn" onclick="document.getElementById('date-picker-example').toggleAttribute('hidden')">Show/Hide</button>
+
+    <div id="date-picker-example" class="date-picker">
+        <div class="calendar-month">
```

Comment:
In BEM, whenever we define a new _block_ (calendar-month is the start of a new block here) there is an expectation that this block is portable and has a use in other situations (e.g it can be used outside of the `date-picker` block). If this is the wish, we should create a new module for this block and show examples of it.

Reply:
I do think a standalone calendar block would be useful in cases where only static/readonly calendar information is required (although there is no design requirement for that right now) - it would also give us a good idea of the responsive behavior of the core calendar piece (e.g. months stacking vertically on smaller screens).

Reply:
Closing as duplicate of https://github.com/eBay/skin/pull/1907#discussion_r1055674698